migrator/migrations/3_add_grant_user_to_users.down.sql

```diff
@@ -0,0 +1 @@
+REVOKE SELECT, INSERT, UPDATE  ON users FROM gateway;
```

Comment:
What about DELETE?

Reply:
By the grant we didn't give this privilege to this user

Reply:
 Why don't we? Delete operation will be required for REST interface and command line tool.

Reply:
fix it